src/kv/kv_types.h

```diff
@@ -354,7 +354,6 @@ namespace kv
     PASS_BACKUP_SIGNATURE_SEND_ACK = 4,
     PASS_NONCES = 5,
     PASS_NEW_VIEW = 6,
-    PASS_SNAPSHOT_EVIDENCE = 7,
```

Comment:
Is that gap correctly handled as unknown status? We don't want it falling through accidentally.

Reply:
Great, thank you.